app/controllers/team.js

```diff
@@ -1,21 +1,28 @@
 import Controller from '@ember/controller';
-import { sort } from '@ember/object/computed';
-import { computed } from '@ember/object';
 
 function inTeam(team) {
-  return computed('sortedModel.[]', function () {
-    return this.sortedModel.filter((member) => member.teams.includes(team));
-  });
+  return (target, key) => {
+    Object.defineProperty(target, key, {
+      get() {
+        return this.sortedModel.filter((member) => member.teams.includes(team));
+      },
+    });
+  };
```

Comment:
I thought your decorator approach was interesting. At a later point, I think we'll end up making a utility function that filters `this.model`, but we can keep this code around for now.